Makefile

```diff
@@ -24,7 +26,7 @@ test:lint
 
 .PHONY: lint
 lint:
```

Comment:
I think you can try to use it for pre-hooks https://git-scm.com/book/ru/v1/%D0%9D%D0%B0%D1%81%D1%82%D1%80%D0%BE%D0%B9%D0%BA%D0%B0-Git-%D0%9F%D0%B5%D1%80%D0%B5%D1%85%D0%B2%D0%B0%D1%82%D1%87%D0%B8%D0%BA%D0%B8-%D0%B2-Git